app/services/submissions_inviter.rb

```diff
@@ -0,0 +1,20 @@
+class SubmissionsInviter
+  def initialize(submission_repository: SubmissionRepository.new)
+    @submission_repository = submission_repository
+  end
```

Comment:
I like that you use dependency injection here and in `SubmissionsExpirationHandler` 👍